docs/migration-pulp-installer-to-multi-process.md

```diff
@@ -0,0 +1,286 @@
+
+## Limitations
+
+1. All of your existing installed plugins must be installed in the multi-process container image. See the list of installed plugins [here.](multi-process-images#available-images)
+
+## Prerequisites
+
+1. Either [podman](https://podman.io/getting-started/installation) or
+[docker](https://docs.docker.com/engine/install/)/[moby-engine](https://mobyproject.org/)
+must be installed on the host.
+2. If you are running podman in rootless mode (which is recommended for security), make sure your user account has subuid's and subgid's (the default behavior for new acconts on many Linux distros.) See [this guide](https://github.com/containers/podman/blob/main/docs/tutorials/rootless_tutorial.md#etcsubuid-and-etcsubgid-configuration) to verify / set this up.
+3. If you are running docker/moby-engine, either the user account is in the docker group, or you preface `docker` commands with `sudo`.
+
+## Assumptions
+
+1. The directory you run the commands in is one where you want the Pulp data and configuration directories to reside under. You can actually let them reside in their current (pulp_installer defaulted or specified) directories on the system if you'd prefer, just specify the absolute folder paths in the commands below, and do not run the move commands.
+2. If you are running rootless podman, you are running `podman` commands from the account that the container will run under. 
+3. sudo commands are run under an account that actually has sudo. This need not necessarily be the rootless podman account, but if it isn't, substitute `$USER:$(id -gn)` with the user and primary group (seperated by a colon) of the rootless podman account.
+4. If you are running docker, substitue `docker` for `podman` in the commands.
+
+## Step-By-Step-Guide
+
+### Disable and stop the Pulp service
+
+```
+sudo systemctl disable --now pulpcore
+```
+
+### Configure PostgreSQL to listen on all network interfaces (Fedora 36 and Ubuntu 22.04 only)
+
+This is necessary for container networks to access PostgreSQL.
+
+Ensure this line is in `/var/lib/pgsql/data/pg_hba.conf` (Fedora 36) / `/etc/postgresql/14/main/pg_hba.conf` (Ubuntu 22.04):
+```
+host all all 0.0.0.0/0 md5
+```
+
+Ensure this line is in `/var/lib/pgsql/data/postgresql.conf` (Fedora 36) / `/etc/postgresql/14/main/postgresql.conf` (Ubuntu 22.04):
+```
+listen_addresses = '*'
+```
+
+Run:
+```
+sudo systemctl restart postgresql
+```
+
+### Dump the PostgreSQL database (EL7 and EL8 only)
+
+```
+sudo -u postgres pg_dumpall > /tmp/dump.sql
+```
+
+Verify that this command outputs "0", which indicates that the dump was successful.
+
+```
+echo $?
+```
+
+```
+sudo mv /tmp/dump.sql /var/lib/pgsql
+```
+
+### Disable/restart the third-party services
+
+On Fedora 36:
+```
+sudo systemctl disable --now nginx redis
+```
+On Ubuntu 22.04:
+```
+sudo systemctl disable --now nginx redis-server
+```
+On on EL, or Fedora prior to 36:
+```
+sudo systemctl disable --now postgresql nginx redis
```

Comment:
Why disable `postrgresql` if we configured it to listen on all interfaces and accept connections from all hosts?

Reply:
As we discussed, we do that on different distros.